Makefile

```diff
@@ -1,6 +1,10 @@
 PREFIX            ?= $(shell pwd)
 FILES_TO_FMT      ?= $(shell find . -path ./vendor -prune -o -name '*.go' -print)
 
+# Ensure everything works even if GOPATH is not set, which is often the case.
+# Default to standard GOPATH.
+GOPATH            ?= $(HOME)/go
```

Comment:
I would say it is `$(shell go env GOPATH)` might be better. What do you think? (:

Reply:
Yep, this looks better. Except it doesn't work with go 1.6 and 1.7. Only from 1.8 onwards. I guess this isn't a concern since we require modules anyway?

Reply:
Yes, we can aim for go1.12+